<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMenuMac</class>
 <widget class="QWidget" name="FormMenuMac">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormMenuMac</string>
  </property>
  <property name="styleSheet">
   <string notr="true">FormMenuMac {
background: gray;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>基本</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>网络</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1024</width>
     <height>590</height>
    </rect>
   </property>
   <widget class="FormMenuMac1" name="page_1"/>
   <widget class="FormMenuMac2" name="page_2"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormMenuMac1</class>
   <extends>QWidget</extends>
   <header>FormMenuMac1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuMac2</class>
   <extends>QWidget</extends>
   <header>FormMenuMac2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
